### DISCUSSION TITLE
exception для определенных чартов через cli

### DISCUSSION THREAD

**Discussion by patsevanton** [helmwave/helmwave#932] [Q&A] [answered]
Привет! Спасибо за helmwave
Я пытаюсь использовать helmwave в установке sentry https://github.com/patsevanton/sentry-strimzi-kafka-clickhouse-operator
Там вот такой workflow
```
# Установка zookeeper через helm
helmwave up --build

# Установка Clickhouse через yaml файл
kubectl apply -f kind-ClickHouseInstallation.yaml

# Расскоментируем sentry в helmwave.yml
helmwave up --build
```

Можно ли запустить используя helmwave установку yaml файлов?
Можно ли запустить helmwave up --build c exception для определенных чартов через cli ?

**Comment by r3nic1e** [comment, marked as answer]
Ямлики без чарта установит через helmwave нельзя

Для выбора, какие чарты надо ставить, в helmwave есть [механизм тегов](https://docs.helmwave.app/0.38.x/examples/tags/)
